.github/workflows/verify.yaml

```diff
@@ -18,12 +18,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: '0'
+
       - uses: ./.github/actions/setup
 
       - name: Run golangci-lint
-        uses: golangci/golangci-lint-action@v6
+        uses: golangci/golangci-lint-action@v8
         with:
-          args: --timeout=10m
+          args: --new-from-rev=HEAD~ --timeout=10m
```

Comment:
Where will we find about them then ?

I know it has always been painful (having `gosec` specifically in mind) that you cannot PR something anymore because you first have to fix new findings due to a tool update, but if we would not be that strict I would assume nobody takes care of that.

Reply:
I can open an issue with the existing findings and we can discuss them in the community meeting?

Reply:
That is just one side of it. Your change will for all time cause new findings due to tooling changes to be hidden. For that community meeting would be a good place to decide whether that is a good idea and how to handle that.

Reply:
> That is just one side of it. Your change will for all time cause new findings due to tooling changes to be hidden. For that community meeting would be a good place to decide whether that is a good idea and how to handle that.

I am not sure I follow. That flag will only show linter findings that are introduced with the respective PR. So in theory the amount of current findings in the code remains the same. New findings in new PR should then be addressed in the respective PR.

Reply:
I am hunting after those new findings that are in existing code but found because the validation tool is enhanced. Like recently when `gosec` added integer casting validation. From one moment to the next one when they released that, all new/updated pull requests became red because they had those findings. We then had to fix those first and rebased the other PRs. If we tell the tool to only bring up findings in modified code, then we will not notice those new "tool update findings" anymore. And that we should decide as community. Imo we should either not do what you suggest, or we require some replacement - could be a daily action that scans the existing code and opens an issue when there are findings (like we do with vulns in the latest release).